app/services/articles/enrich_image_attributes.rb

```diff
@@ -37,22 +34,14 @@ def self.call(article)
 
         next if image.blank?
 
-        attribute_width, attribute_height = image_width_height(img)
-        img["width"] = attribute_width
-        img["height"] = attribute_height
-        img["data-animated"] = true if FastImage.animated?(image, timeout: TIMEOUT, raise_on_failure: false)
+        property = FastImage.new(image, timeout: 10)
+        img["width"], img["height"] = property.size
+        img["data-animated"] = true if property.type == :gif
```

Comment:
The idea here is that checking for `.gif` is good enough. We don't need to be completely accurate by checking for frame counts with `FastImage.animated?`

Reply:
I guess this wouldn't catch animated images other than gifs, right (like webp)? I'm wondering because I have some vague memory that we do some image manipulation on forem cloud to convert gifs to webp, and I'm curious if these would still be caught by this 🤔  I could be remembering all wrong though 😂 

I'm currently working on https://github.com/forem/rfcs/pull/188 so keen to know how this change might reduce scope of what we're catching.

Reply:
Ah you are right. I'm going to have to revert this change then.

Reply:
Well, this is a bit tough. Looks like the FastImage only check for the animated frame if it's a `:gif`
https://github.com/sdsykes/fastimage/blob/6478cd33aca80982363b1043680d3d31c79b0ac9/lib/fastimage.rb#L432-L435

We can't just assume that all webp is animated, is that correct?

Reply:
TIL webp images can be animated

Reply:
> Looks like the FastImage only check for the animated frame if it's a :gif

😱  TIL!

I don't think we can assume all webp is animated... I guess given the constraints with FastImage, we might just need to limit our play/pause ability to `.gif` 🤷‍♀️   I agree that at least for now, what you have here is the way forward.

TBH I think we could probably live with only making `.gif`s play/pausable (seems fairly reasonable to me), but the issue becomes if we ourselves convert `gif` to `webp` and then can't identify this `data-animated` property as a result 🙃

Reply:
Once we arrive at a solution, later on, we can re-process all the markdowns and have `data-animated` properly appear where it should